1-Draft/RFC00XX-Update-Error-View.md

```diff
@@ -0,0 +1,204 @@
+---
+RFC: Update $ErrorView with simplified views for Error Messages
+Author: Jason Helmick
+Status: Draft
+Version: 0.0.0
+Area: PowerShell
+Comments Due: 10/31/2019
+---
+
+# Update $ErrorView with simplified views for Error Messages
+
+When an error occurs in PowerShell, the customers on-screen error message experience currently
+provides a level of detail that obscures the exception message from being recognized and read by
+new or occasional PowerShell users. The addition of a simplified error view will improve both
+comprehension and troubleshooting experience. A new cmdlet 'Resolve-ErrorRecord' will provide
+complete detailed view of the fully qualified error when desired.
+
+## Motivation
+
+The on-screen experience, when receiving an error message,
+is controlled through the views NormalView (the default) and CategoryView. These are user selectable
+through the preference variable $ErrorView.
+This RFC describes Changing '$ErrorView to an enumeration and adding three additional views
+to improve readability,'Message', 'Analytic', and 'Dynamic'.
```

Comment:
I don't see any examples of setting $ErrorView. Will $null and 'CategoryView' still be valid? In fact, after a few seconds of testing, it appears that Windows PS 5.1 does no validation of $ErrorView except to check whether it's 'CategoryView' (case-insensitive) before printing an error message. So it appears that there is some possibility of a breaking change.

Reply:
@essentialexch you are correct that there is currently no validation and that this change may be breaking.  I will include some examples of setting the errorview, however I'm thinking it will be the same as today. $errorview = 'ViewName'. Also, I will be updating the view names to help lessen the impact of a breaking change.